static/styles/dark_theme.css

```diff
@@ -58,6 +60,22 @@ body.dark-theme {
         box-shadow: inset 0 1px 0 hsla(0, 0%, 0%, 0.2);
     }
 
+    .messagebox {
+        background-color: hsl(0, 0%, 14%);
+        &:hover {
+            /* Overlay a dark filter over messages on hover, by changing the background of the div over it. */
```

Comment:
For light theme I added the hover colours individually, and here I did it this way because it was faster and easier -- this feels nice because it keeps the overlay consistent, but it also feels a bit sketch because it assumes another div is in a certain place. Thoughts?

Reply:
It depends on what effect you want on the border of `messagebox`. Other than that, this should work fine.